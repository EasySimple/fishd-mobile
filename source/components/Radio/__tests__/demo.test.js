import webDemoTest from '../../../tests/demoTest';

webDemoTest('Radio');
