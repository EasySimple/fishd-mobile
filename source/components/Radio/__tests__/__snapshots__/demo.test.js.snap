// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Radio demo correctly 1`] = `
<div>
  <div
    class="components-radio-demo-basic"
  >
    <div
      class="sub-title"
    >
      基础样式
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          checked=""
          class="fm-list-item fm-radio-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio fm-radio-checked"
              >
                <input
                  checked=""
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项一
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
        <div
          class="fm-list-item fm-radio-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio"
              >
                <input
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项二
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-radio-demo-basic"
  >
    <div
      class="sub-title"
    >
      水平排列
    </div>
    <div
      class="fm-list m-radio-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          checked=""
          class="fm-list-item fm-radio-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio fm-radio-checked"
              >
                <input
                  checked=""
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项一
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
        <div
          class="fm-list-item fm-radio-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio"
              >
                <input
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项二
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-radio-demo-basic"
  >
    <div
      class="sub-title"
    >
      禁用状态
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          checked=""
          class="fm-list-item fm-radio-item fm-radio-item-disabled fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio fm-radio-checked fm-radio-disabled"
              >
                <input
                  checked=""
                  class="fm-radio-input"
                  disabled=""
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项一
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
        <div
          class="fm-list-item fm-radio-item fm-radio-item-disabled fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio fm-radio-disabled"
              >
                <input
                  class="fm-radio-input"
                  disabled=""
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项二
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-radio-demo-basic"
  >
    <div
      class="sub-title"
    >
      自定义颜色
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          checked=""
          checkedcolor="#07c160"
          class="fm-list-item fm-radio-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio fm-radio-checked"
              >
                <input
                  checked=""
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                  style="background-color:#07c160;border-color:#07c160"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项一
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
        <div
          checkedcolor="#07c160"
          class="fm-list-item fm-radio-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio"
              >
                <input
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项二
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-radio-demo-basic"
  >
    <div
      class="sub-title"
    >
      自定义形状
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          checked=""
          class="fm-list-item fm-radio-item fm-list-item-middle"
          shape="square"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio fm-radio-checked"
              >
                <input
                  checked=""
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                  style="border-radius:3px"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项一
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
        <div
          class="fm-list-item fm-radio-item fm-list-item-middle"
          shape="square"
        >
          <div
            class="fm-list-thumb"
          >
            <label
              class="fm-radio-wrapper"
            >
              <span
                class="fm-radio"
              >
                <input
                  class="fm-radio-input"
                  type="radio"
                />
                <span
                  class="fm-radio-inner"
                  style="border-radius:3px"
                />
              </span>
            </label>
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              选项二
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
